src/main/java/frc/robot/auto/AutoSelector.java

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
Ideally we wouldn't have non-swerve drive related changes inside this commit. This ensures we're able to keep commits small and manageable, as well as understand what all the changes relate to.